statik/README.md

```diff
@@ -0,0 +1,20 @@
+### Regenerating the Admin UI Zip Blob
+
+Please note that this file is automatically generated by the
+[statik](https://github.com/rakyll/statik) tool, and should not be
+updated directly. It is generated from the contents of the
+`/shared/admin` directory, so make updates at that location first, and
+then continue with this guide.
+
+If changes are made to the InfluxDB Admin UI, then this file can be
+regenerated using the following commands:
+
+```
+# Assuming the InfluxDB repo is located at `~/gocodes/src/github.com/influxdb/influxdb`
```

Comment:
Probably want to use the `github.com/influxdata/influxdb` path in these examples.